lms/djangoapps/teams/static/teams/js/views/instructor_tools.js

```diff
@@ -25,14 +29,46 @@
 
                 deleteTeam: function (event) {
                     event.preventDefault();
-                    alert("You clicked the button!");
-                    //placeholder; will route to delete team page
+                    var self = this;
+                    var msg = new PromptView.Warning({
+                        title: gettext('Delete this team?'),
+                        message: gettext('Deleting a team removes it from the team listing view, and removes members from the team as well.'),
+                        actions: {
+                            primary: {
+                                text: gettext('Delete'),
+                                class: 'action-delete',
+                                click: function () {
+                                    msg.hide();
+                                    self.handleDelete();
+                                }
+                            },
+                            secondary: {
+                                text: gettext('Close'),
+                                class: 'action-cancel',
+                                click: function () {
+                                    msg.hide();
+                                }
+                            }
+                        }
+                    });
+                    msg.show()
                 },
 
                 editMembership: function (event) {
                     event.preventDefault();
                     alert("You clicked the button!");
                     //placeholder; will route to remove team member page
+                },
+
+                handleDelete: function () {
+                    var self = this;
+                    this.team.destroy().then(function () {
```

Comment:
I'm thinking that you should have error handling in case this team can't be deleted for some reason. We could rely on the general purposes AJAX error handling that we have to do in a subsequent story, but I'm thinking we could give more targeted feedback here.